app/components/header.hbs

```diff
@@ -58,25 +58,7 @@
             </div>
           </FeedbackButton>
 
-          {{! TODO: Extract this into BillingStatusBadgeComponent, re-use in course page header and here? }}
-          {{#if this.billingStatusDisplay.shouldShowVipBadge}}
-            <Header::VipBadge class="mr-3" />
-          {{else if this.billingStatusDisplay.shouldShowMemberBadge}}
-            <Header::MemberBadge class="mr-3" />
-          {{else if this.activeDiscountForYearlyPlan}}
-            {{#if this.canSeeDiscountCountdown}}
-              <BillingStatusBadge::DiscountTimerBadge @discount={{this.activeDiscountForYearlyPlan}} @size="small" class="mr-3" />
-            {{/if}}
-          {{else if this.billingStatusDisplay.shouldShowFreeWeeksLeftButton}}
-            <Header::FreeWeeksLeftButton class="mr-3" />
-          {{else if this.billingStatusDisplay.shouldShowUpgradeButton}}
-            <PrimaryLinkButton @size="extra-small" @route="pay" class="mr-4" data-test-upgrade-button>
-              <span class="flex items-center gap-x-1">
-                <span>Upgrade</span>
-                {{svg-jar "lock-open" class="w-4"}}
-              </span>
-            </PrimaryLinkButton>
-          {{/if}}
+          <BillingStatusDisplay::BillingStatusBadge @size="small" />
```

Comment:
```suggestion
          <BillingStatusDisplay::BillingStatusBadge @size="small" class="mr-3" />
```

(This is the correct place to handle margins)

The `mr-4` that was previously applied to the Upgrade button above seems like a mistake, I think it's okay to remain consistent and use `mr-3` for all badges.